odiglet/pkg/kube/instrumentation_ebpf/pods.go

```diff
@@ -25,6 +26,11 @@ type PodsReconciler struct {
 
 func (p *PodsReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
+
+	if request.Namespace == consts.DefaultOdigosNamespace {
```

Comment:
better to use `env.GetCurrentNamespace()` to support non `odigos-system` namespaces